app/models/calculation_result.rb

```diff
@@ -148,13 +157,21 @@ def client_capital_rows
   end
 
   def client_capital_subtotal_rows
-    rows = {
-      total_capital: monetise(api_response.dig(:result_summary, :capital, :total_capital)),
-      pensioner_capital_disregard: monetise(-api_response.dig(:result_summary, :capital, :pensioner_capital_disregard)),
-      smod_disregard: monetise(-api_response.dig(:result_summary, :capital, :subject_matter_of_dispute_disregard)),
-    }
-
-    if has_partner? && pensioner_disregard_applied?
+    rows = if has_partner?
```

Comment:
I think this `if` statement is based on a slight misunderstanding of the ticket: there should never be a situation where pensioner capital disregard appears above smod disregard. The ticket is saying "if pensioner disregard is shown _at all_ in this table, show it beneath smod".